piper.in

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+import sys
+import signal
+import os
+import locale
+import gettext
+
+import argparse
+import logging
+from piper import piper
+
+import gi
+gi.require_version('Gio', '2.0')
+gi.require_version('Gtk', '3.0')
+from gi.repository import Gio, Gtk
+
+localedir = '@localedir@'
```

Comment:
Perhaps this has to be set in Meson, I haven't looked into that yet. Doesn't really matter for now, as there are no translations yet anyway.